app/javascript/src/Logger.js

```diff
@@ -0,0 +1,34 @@
+import { Logtail } from '@logtail/browser';
+
+export default class Logger {
+  #environment;
+  #gameId;
+
+  #logtail;
+
+  constructor(environment, gameId) {
+    this.#environment = environment;
+    this.#gameId = gameId;
+
+    this.#logtail = new Logtail('3bdHcA8P3mcww2ojgC5G8YiT');
+  }
+
+  error(message, data) {
+    data.gameId = this.#gameId;
+
+    switch (this.#environment) {
+      case 'production':
+        this.#logtail.error(message, data);
+        break;
+
+      case 'replay':
+      case 'unittests':
```

Comment:
I don't imagine we'll want to use the logger in the test environment (as per my comments above, too).

Reply:
[76f61f0](https://github.com/Thrillberg/imperial/pull/729/commits/76f61f00f58e876e15be9e5e2d7e0ddfdbd8a432)